Update extraRpcs.js

say-malove opened the pull request:
Adding two new public endpoints (Arb and OP), and updating privacy. Our privacy policy is here - https://drpc.org/privacy-policy

If you are adding a new RPC, please answer the following questions.

#### Link the service provider's website (the company/protocol/individual providing the RPC):
https://drpc.org/

#### Provide a link to your privacy policy:
https://drpc.org/privacy-policy

#### If the RPC has none of the above and you still think it should be added, please explain why:

Your RPC should always be added at the end of the array.

realdealshaman commented:
@say-malove thanks so much for the follow up. We would need a sentence we can cite from the privacy policy where it states that you do not store the IP address of users. I was only able to find this:

say-malove commented:
@realdealshaman Hi, thanks for your reply, we have updated the privacy policy and added specific timings - https://drpc.org/privacy-policy.

You can check under - Cookies, Logs, and Pixels
